source/routing/raptor_api.cpp

```diff
@@ -980,14 +980,36 @@ void free_radius_filter(routing::map_stop_point_duration& sp_list,
     }
 }
 
+DateTime prepare_next_call_for_raptor(const RAPTOR::Journeys& journeys, const bool clockwise)
+{
+    DateTime lastest_arrival    = DateTimeUtils::min;
+    DateTime earliest_departure = DateTimeUtils::inf; // clockwise
```

Comment:
minor: this code is a bit weird to declare 2 variables and use only one...
Maybe a single `datetime_next` variable, or always update the 2 variables and use only one at the end?

Reply:
:heavy_check_mark: